### DISCUSSION TITLE
Use proxy server for making requests

### DISCUSSION THREAD

**Discussion by jgarrow** [PaloAltoNetworks/docusaurus-openapi-docs#509] [Q&A]
I'm running into CORS issues when trying to send requests with the "Send API request" button. I'm fairly confident that the API I'm trying to hit won't allow requests from the browser. Looking in the source code here, I see that [`makeRequest` accepts a `proxy` argument](https://github.com/PaloAltoNetworks/docusaurus-openapi-docs/blob/main/packages/docusaurus-theme-openapi-docs/src/theme/ApiDemoPanel/Execute/makeRequest.ts#L49). How do I set up/configure a proxy server with Docusaurus and tell the `makeRequest` function to use it?

**Comment by sserrata** [comment]
Check the docs: https://docusaurus-openapi.tryingpan.dev/#config

There's a `proxy` option. 

For reference, the Petstore demo implements this feature.
